app/src/main/java/ch/epfl/sweng/vanjel/nearbyDoctor/ListNearbyDoctorsAdapter.java

```diff
@@ -85,14 +84,15 @@ public int getItemCount() {
 
     public class ViewHolder extends RecyclerView.ViewHolder {
 
-        final TextView firstName, lastName, activity, distance;
+        TextView address, lastName, activity, distance;
```

Comment:
The final might have been added by @ElCabrino in order to reduce warning. @ElCabrino can you confirm this?